### DISCUSSION TITLE
Can't Deploy Without Adding PR Creator To Vercel Team

### DISCUSSION THREAD

**Discussion by armordog** [vercel/vercel#6493] [Help] [answered]
Why does Vercel need to know (or care) who made the PR on Github?
It's just supposed to build commits to the defined branch.

But anyone who makes a PR gets this message:
![image](https://user-images.githubusercontent.com/1992702/126260071-82ccbc5c-1f3e-4660-b4ea-a3ef64e2ab21.png)

> {user} is attempting to deploy a commit to the {team} Team on Vercel.
> To accomplish this, {user} needs to request access to the Team.
> Afterwards, an owner of the Team is required to accept their membership request.

**Comment by leerob** [comment, marked as answer]
Hey! Vercel teams require all members of the git repository to be members of the Vercel team 😄 

https://vercel.com/docs/git#using-teams

**Reply by armordog** [reply]
Ah, so it's just a part of Vercel's pricing model, I see.
Thank you.
